docs/source/gdk-json.rst

```diff
@@ -666,16 +666,48 @@ Receive Address JSON
 
 
 
-.. _unspent-utxos-details:
+.. _unspent-utxos-request:
```

Comment:
nit: if we're in the business of renaming this, can we call it 'unspent-outputs-request'? The would make it consistent with the name of the function, plus 'unspent utxos' is a bit tautological.

Reply:
Good point, renamed.